latch_cli/click_utils.py

```diff
@@ -51,7 +143,84 @@ def format_epilog(self, ctx: Context, formatter: HelpFormatter) -> None:
         )
 
 
-class LatchGroup(LatchCommand, Group): ...
+
+----
+class EnumChoice(Choice):
+    def __init__(self, choices: Type[Enum], case_sensitive: bool = True):
+        self.enum = choices
+        return super().__init__(choices._member_names_, case_sensitive)
+
+    def convert(
+        self, value: Any, param: Optional[Parameter], ctx: Optional[Context]
+    ) -> Any:
+        ret = super().convert(value, param, ctx)
+        return self.enum(ret)
+
+    # todo(ayush): override `shell_complete` once we support it
+
+
+class ColoredHelpFormatter(HelpFormatter):
+    def write_usage(
+        self, prog: str, args: str = "", prefix: Optional[str] = None
+    ) -> None:
+        if prefix is None:
+            prefix = "Usage: "
+
+        prefix = style(prefix, fg="bright_blue")
+        return super().write_usage(prog, args, prefix)
+
+    def write_heading(self, heading: str):
+        return super().write_heading(style(heading, fg="bright_blue"))
+
+    def write_dl(
+        self, rows: Sequence[Tuple[str, str]], col_max: int = 30, col_spacing: int = 2
+    ) -> None:
+        return super().write_dl(
+            [(style(x[0], fg="bright_green", italic=True), x[1]) for x in rows],
+            col_max,
+            col_spacing,
+        )
+
+
+class LatchCommand(Command):
+    def format_epilog(self, ctx: Context, formatter: HelpFormatter) -> None:
+        formatter.write_paragraph()
```

Comment:
i cant see any differences between these and the ones above - why are these being copy-pasted here? also why the `----`?